Library/Homebrew/formulary.rb

```diff
@@ -183,6 +183,7 @@ def get_formula(spec)
   class NullLoader < FormulaLoader
     def initialize(name)
       @name = name
+      @path = nil
```

Comment:
Let all `FormulaLoader` instances have `path` attribute. And `nil` means formula unavailable.